Lib/locale.py

```diff
@@ -279,6 +279,8 @@ def currency(val, symbol=True, grouping=False, international=False):
         if precedes:
             s = smb + (separated and ' ' or '') + s
         else:
+            if international and smb[-1] == ' ':
+                smb = smb[:-1]
```

Comment:
why don't use `rstrip()?`

Reply:
To be more specific about the case (1 trailing space).